cmd/deviceplugin/main.go

```diff
@@ -157,3 +148,13 @@ func main() {
 	}
 	select {}
 }
+
+func defaultResName() string {
+	devResourceName, ok := os.LookupEnv(EnvVarResourceName)
+	if !ok {
+		klog.Fatalf("no resource name configured - nothing to do")
```

Comment:
this will make the plugin exits uncleanly. Which is ok, but then the change qualifies as change of behavior, not as refactoring/cleanup

Reply:
Do you mean because in the original code it exists with `0`?

Reply:
yes indeed!